code/modules/clothing/under/miscellaneous.dm

```diff
@@ -693,13 +766,15 @@
 	item_color = "lobster"
 	fitted = NO_FEMALE_UNIFORM
 	can_adjust = FALSE
+
 /obj/item/clothing/under/gondola
 	name = "gondola hide suit"
 	desc = "Now you're cooking."
 	icon_state = "gondola"
 	item_state = "lb_suit"
 	item_color = "gondola"
 	can_adjust = FALSE
+
```

Comment:
Lot of whitespaces, sure, but it's very irritating how the paths lacked spacing from one other, so ok.

Reply:
They were removed somewhat and lots of random white space were old flags were got removed.
But I frankly dont know why this file and only this one had no white spaces were new items were